src/Interactive/Host/Interactive/Core/InteractiveHost.cs

```diff
@@ -175,36 +152,23 @@ private RemoteService TryStartProcess(string hostPath, CultureInfo culture, Canc
                     newProcessId = 0;
                 }
 
-                // sync:
-                while (!semaphore.WaitOne(_millisecondsTimeout))
-                {
-                    if (!CheckAlive(newProcess, hostPath))
-                    {
-                        return null;
-                    }
-
-                    WriteOutputInBackground(isError: false, string.Format(InteractiveHostResources.Attempt_to_connect_to_process_Sharp_0_failed_retrying, newProcessId));
-                    cancellationToken.ThrowIfCancellationRequested();
-                }
+                // (miziga) initializing pipe for replacement of remote channel
+                NamedPipeClientStream _clientStream = new NamedPipeClientStream(".", pipeName, PipeDirection.InOut, PipeOptions.Asynchronous);
+                JsonRpc jsonRpc = JsonRpc.Attach(_clientStream);
```

Comment:
Let's move this under the `try` below, so that we catch any exceptions that might happen during the pipe creation.